edge-agent/README.md

```diff
@@ -14,7 +14,8 @@ The application will read the `appsettings.json` for configuration. The format i
 {
   "DockerUri": "<docker service uri>",
   "DeviceConnectionString": "<Your IoT Hub Device Connection String>",
-  "ConfigSource": "<iothubconnected|standalone>",
+  "ConfigSource": "<twin|local>",
+  "ConfigSourceBindPath": "<Bind path to the folder where the config.json is in 'local' mode>",
```

Comment:
Nit: suggest renaming to LocalConfigPath

Reply:
Done